TELCODOCS#2164: Added pin information (#95044)
openshift/openshift-docs: modules/nw-ptp-dual-wpc-hardware-config-reference.adoc

|`spec.profile.ptp4lConf`
|Set the value of `boundary_clock_jbod` to 1 to enable support for multiple NICs.
|====

[NOTE]
====
Each value in the `spec.profile.plugins.e810.pins` list follows the format `<function>` `<connector ID>`, where:

* `<function>` specifies the pin role:

** `0` = Disabled
** `1` = Receive (Rx) – for 1PPS IN
** `2` = Transmit (Tx) – for 1PPS OUT

* `<connector ID>` identifies the physical connector:

** `1` = `SMA1` or `U.FL1`
** `2` = `SMA2` or `U.FL2`

Examples:

* `2 1` enables `1PPS OUT` (Tx) on `SMA1`.

> * `1 1` enables `1PPS IN` (Rx) on `SMA1`.

Reviewer: same as above

> * `<function>` specifies the pin role:
> 
> ** `0` = Disabled
> ** `1` = Receive (Rx) – for 1PPS IN
> ** `2` = Transmit (Tx) – for 1PPS OUT
> 
> * `<connector ID>` identifies the physical connector:
> 
> ** `1` = `SMA1` or `U.FL1`
> ** `2` = `SMA2` or `U.FL2`
> 
> Examples:
> 
> * `2 1` enables `1PPS OUT` (Tx) on `SMA1`.
> * `1 1` enables `1PPS IN` (Rx) on `SMA1`.

Reviewer 2: ```suggestion
`<function>`:: Specifies the pin role.
+
The following values are associated with the pin role:
+
`0`::: Disabled.
`1`::: Receive (Rx) - for 1PPS IN.
`2`::: Transmit (Tx) - for 1PPS OUT.

`<connector ID>`:: Identifies the physical connector.
+
The following values are associated with the physical connector:
+
`1`::: `SMA1` or `U.FL1`.
`2`::: `SMA2` or `U.FL2`.

Examples::
`2 1`::: Enables `1PPS OUT` (Tx) on `SMA1`.
`1 1`::: Enables `1PPS IN` (Rx) on `SMA1`.
```

Reviewer 2: https://docs.asciidoctor.org/asciidoc/latest/lists/description/

Reviewer 2: https://www.ibm.com/docs/en/ibm-style?topic=format-lists#definition-lists

> These values are passed through the PTP Operator to the Intel E810 hardware plugin and eventually written to the sysfs pin configuration interface on each NIC.

Reviewer 2: ```suggestion
The PTP Operator passes these values to the Intel E810 hardware plugin, and then the Intel E810 hardware plugin writes these values to the sysfs pin configuration interface on each NIC.
```
